providers/org/elife/harvester.py

```diff
@@ -1,5 +1,97 @@
+import logging
+
+from furl import furl
+
+from itertools import chain
+
+from lxml import etree
+
 from share.harvest.harvester import Harvester
 
 
+logger = logging.getLogger(__name__)
+
+
 class ELifeHarvester(Harvester):
```

Comment:
While its fairly generous make sure to include githubs rate limit https://developer.github.com/v3/#rate-limiting